clientLibrary/src/main/scala/za/co/absa/clientexample/Application.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.clientexample
+
+import za.co.absa.clientexample.config.ConfigProvider
+import za.co.absa.loginclient.authorization.JWTDecoderProvider
+import za.co.absa.loginclient.tokenRetrieval.service.RetrieveToken
+
+import java.util.Scanner
+
+object Application {
+  def main(args: Array[String]): Unit = {
+
+    val config = new ConfigProvider("clientLibrary/src/main/resources/exampleConfig.yaml").
```

Comment:
I think this should be overridable from the `args` app params

Reply:
Done